client/state/jetpack-connect/actions.js

```diff
@@ -63,6 +64,15 @@ const remoteAuthPath =
 const remoteInstallPath = '/wp-admin/plugin-install.php?tab=plugin-information&plugin=jetpack';
 const remoteActivatePath = '/wp-admin/plugins.php';
 
+const persistSession = ( url, flowType ) => {
+	const options = {
+		maxAge: 300,
```

Comment:
This value (seconds) needs to match up with the old `isStale()` check from the selector. Add a constant.